Resources/Prototypes/Loadouts/head.yml

```diff
@@ -107,6 +155,14 @@
   items:
     - ClothingHeadHatBellhop
 
+- type: loadout
+  id: LoadoutHeadFlower
+  category: Head
+  cost: 1
+  exclusive: true
+  items:
+    - ClothingHeadHatHairflower
```

Comment:
```suggestion
```

Hairflower got removed in a recent update, but now you can just wear a poppy on your head that has exactly the same appearance as before.

Reply:
Why? Can we re-add that?

Reply:
resolved